helm/portieris/templates/webhooks.yaml

```diff
@@ -1,12 +1,54 @@
+# webhook which is inoperative (no match in namespaceSelector) but in the helm manifest such that it it uninstalled on chart delete
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
   name: image-admission-config
+  annotations:
   {{ if .Values.UseCertManager }}
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/portieris-certs
+  {{ end }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "portieris.name" . }}
+    chart: {{ template "portieris.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+webhooks:
+  - name: trust.hooks.securityenforcement.admission.cloud.ibm.com
+    clientConfig:
+      service:
+        name: {{ template "portieris.name" . }}
+        namespace: {{ .Release.Namespace }}
+        path: "/admit"
+      {{ if not .Values.UseCertManager }}
+      caBundle: {{ .Files.Get "certs/ca.crt" | b64enc }}
+      {{ end }}
+    rules:
+      - operations: [ "CREATE", "UPDATE" ]
+        apiGroups: ["*"]
+        apiVersions: ["*"]
+        resources: ["pods", "deployments", "replicationcontrollers", "replicasets", "daemonsets", "statefulsets", "jobs", "cronjobs"]
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions: ["v1beta1"]
+    namespaceSelector:
+      matchExpressions:
+      - key: securityenforcement.admission.cloud.ibm.com/namespace
+        operator: In
+        values:
+        - notExist
+---
+# webhook similar to the above installed by a post-install helm hook to activate (remove or update the namesapceSelector)
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: image-admission-config
   annotations:
-    cert-manager.io/inject-ca-from: {{ .Values.namespace }}/portieris-certs
+    "helm.sh/hook": post-install,post-upgrade,post-rollback
+  {{ if .Values.UseCertManager }}
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/portieris-certs
   {{ end }}
-  namespace: {{ .Values.namespace }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
MutatingWebhookConfiguration objects aren't namespaced so this line has no effect and can be removed.